app/views/hq/sponsors/_index.html.haml

```diff
@@ -0,0 +1,43 @@
+- if sponsors.empty?
+  .table-responsive
+    %table.table
+      %tbody
+        %tr
+          %td
+            No Sponsors found
```

Comment:
Much easier to match the styling between the two when they're the same thing.  I think we still need eye candy stories for most of this anyway, this just moves it from terrible to acceptable before those get tackled.  So, yes, I think it's a good idea at this point.  In the finished product, defintely not.  But right now I'm happy if the funcitonality is added while eye candies are pending